care/facility/models/inventory.py

```diff
@@ -0,0 +1,98 @@
+from django.contrib.auth import get_user_model
+from django.core.validators import MinValueValidator
+from django.db import models
+from partial_index import PQ, PartialIndex
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'partial_index'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5683989)